web/models.py

```
@@ -2578,6 +2578,97 @@
+    updated_at = models.DateTimeField(auto_now=True)
+    reviewed_at = models.DateTimeField(null=True, blank=True)
+    reviewed_by = models.ForeignKey(
+        User, on_delete=models.SET_NULL, null=True, blank=True, related_name="reviewed_submissions"
+    )
+
+    class Meta:
+        ordering = ["-submitted_at"]
+        verbose_name = "PDF Submission"
+        verbose_name_plural = "PDF Submissions"
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)